examples/todoapp/package.json

```diff
@@ -7,6 +7,7 @@
                 "assert-plus": "0.1.2",
                 "bunyan": "0.21.1",
```

Comment:
I know this is a demo, but seems like many of the other deps in here are really outdated. Worth revving them too?